concepts/keyword-lists/about.md

```diff
@@ -59,6 +61,8 @@ if age >= 16, do: "beer", else: "no beer"
 
 This may look like `if` accepts two arguments, but the `do:` and `else:` pair is actually a single argument - a keyword list. The same code could be written as:
 
+[]: # (elixir-formatter-disable-next-block)
```

Comment:
Where desired, formatting can be turned off. For example here we want to show off a non-standard way to format this code to prove a point.